### PR TITLE
Utility doc generation

### PR DESCRIPTION
**Pull request by lancetharper**

### CODE REVIEW

File: src/css/deprecated/fall2019/_visibility.scss [hunk 1]
```diff
@@ -3,9 +3,9 @@
   NOTE: these classes are deprecated, and will be removed from Cedar in the Spring 2020 release
 */ 
 
-/*///////////////////////
+/*=======================
 asset v0.1.3 visibility
```
Comment (mhewson) [inline src/css/deprecated/fall2019/_visibility.scss]: Maybe Display: common.css cedar(version)

File: src/css/deprecated/fall2019/_a11y.scss [hunk 2]
```diff
@@ -3,9 +3,9 @@
   NOTE: these classes are deprecated, and will be removed from Cedar in the Spring 2020 release
 */ 
 
-/*///////////////////////
+/*=====================
 asset v0.1.3 ally
```
Comment (mhewson) [inline src/css/deprecated/fall2019/_a11y.scss]: this seems to be the wrong header at this point

**Review by mhewson — DISMISSED** [review]

File: src/css/deprecated/fall2019/_a11y.scss [hunk 3]
```diff
@@ -3,9 +3,9 @@
   NOTE: these classes are deprecated, and will be removed from Cedar in the Spring 2020 release
```
Comment (RatulaREI) [inline src/css/deprecated/fall2019/_a11y.scss]: The "t" of "these" will be uppercase.

File: src/css/deprecated/fall2019/_visibility.scss [hunk 4]
```diff
@@ -3,9 +3,9 @@
   NOTE: these classes are deprecated, and will be removed from Cedar in the Spring 2020 release
```
Comment (RatulaREI) [inline src/css/deprecated/fall2019/_visibility.scss]: The "t" of "these" will be uppercase.

File: src/css/deprecated/fall2019/_a11y.scss (hunk 3 above)
Comment (RatulaREI) [reply src/css/deprecated/fall2019/_a11y.scss]: This is how it should be:
Note: These classes are deprecated, and will be removed from cedar in the Spring 2020 release.

File: src/css/deprecated/fall2019/_visibility.scss (hunk 4 above)
Comment (RatulaREI) [reply src/css/deprecated/fall2019/_visibility.scss]: This is how it should be:
Note: These classes are deprecated, and will be removed from cedar in the Spring 2020 release.

File: src/css/utility/_visibility.scss [hunk 5]
```diff
@@ -18,78 +18,109 @@
...
+*/
+.cdr-display-flex { @include cdr-display-flex-mixin; }
+/*
+/// Sets display to inline-flex. 
+/// Append [@xs, @sm, @md, or @lg] to make responsive.
+/// @group utility.visibility
+*/
+.cdr-display-inline-flex { @include cdr-display-inline-flex-mixin; }
+/*
+/// Screen reader only text.
```
Comment (RatulaREI) [inline src/css/utility/_visibility.scss]: How about "Text for screen reader only"?

File: src/css/utility/_visibility.scss [hunk 6]
```diff
@@ -18,78 +18,109 @@
...
+/// @group utility.visibility
+*/
+.cdr-display-inline-flex { @include cdr-display-inline-flex-mixin; }
+/*
+/// Screen reader only text. 
+/// @group utility.visibility
+*/
+.cdr-display-sr-only { @include cdr-display-sr-only-mixin; }
+/*
+/// Screen reader only text, but focusable.
```
Comment (RatulaREI) [inline src/css/utility/_visibility.scss]: "Text for screen reader only; not visible otherwise." OR, simply "Text for screen reader only."

**Review by RatulaREI — COMMENTED** [review]
Left comments for suggested changes.

**Comment by RatulaREI** [comment]
Left comments of suggested changed. Couldn't make the changes directly.

File: src/css/deprecated/fall2019/_a11y.scss (hunk 2 above)
Comment (lancetharper) [reply src/css/deprecated/fall2019/_a11y.scss]: So...just remove them? I don't know that they really offer any value

File: src/css/deprecated/fall2019/_a11y.scss (hunk 3 above)
Comment (lancetharper) [reply src/css/deprecated/fall2019/_a11y.scss]: These comments don't show up anywhere outside of the code here, but I'll update that anyway.

**Review by cowills — DISMISSED** [review]

**Review by cowills — APPROVED** [review]

**Pull request merged**